apis/placement/v1alpha1/stagedupdate_types.go

```diff
@@ -12,39 +12,41 @@ import (
 )
 
 // +genclient
-// +genclient:namespaced
+// +genclient:Cluster
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:resource:scope="Namespaced",categories={fleet,fleet-placement}
+// +kubebuilder:resource:scope=Cluster,categories={fleet,fleet-placement},shortName=crsur
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// StagedUpdateRun represents a stage by stage update process that applies selected resources to specified clusters.
+// ClusterStagedUpdateRun represents a stage by stage update process that applies ClusterResourcePlacement
+// selected resources to specified clusters.
 // Resources from unselected clusters are removed after all stages in the update strategy are completed.
-// Each StagedUpdateRun object corresponds to a single release of a specific resource version.
-// The release is abandoned if the StagedUpdateRun object is deleted or the scheduling decision changes.
-// The name of the StagedUpdateRun must conform to RFC 1123.
-type StagedUpdateRun struct {
+// Each ClusterStagedUpdateRun object corresponds to a single release of a specific resource version.
+// The release is abandoned if the ClusterStagedUpdateRun object is deleted or the scheduling decision changes.
+// The name of the ClusterStagedUpdateRun must conform to RFC 1123.
+type ClusterStagedUpdateRun struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// The desired state of StagedUpdateRun. The spec is immutable.
+	// The desired state of ClusterStagedUpdateRun. The spec is immutable.
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="The spec field is immutable"
 	Spec StagedUpdateRunSpec `json:"spec"`
 
-	// The observed status of StagedUpdateRun.
+	// The observed status of ClusterStagedUpdateRun.
 	// +kubebuilder:validation:Optional
 	Status StagedUpdateRunStatus `json:"status,omitempty"`
 }
 
 // StagedUpdateRunSpec defines the desired rollout strategy and the snapshot indices of the resources to be updated.
 // It specifies a stage-by-stage update process across selected clusters for the given ResourcePlacement object.
 type StagedUpdateRunSpec struct {
-	// A reference to the placement that this update run is applied to.
+	// PlacementName is the name of placement that this update run is applied to.
 	// There can be multiple active update runs for each placement, but
 	// it's up to the DevOps team to ensure they don't conflict with each other.
 	// +kubebuilder:validation:Required
-	PlacementRef PlacementReference `json:"placementRef"`
+	// +kubebuilder:validation:MaxLength=255
+	PlacementName string `json:"placementName"`
 
 	// The resource snapshot index of the selected resources to be updated across clusters.
 	// The index represents a group of resource snapshots that includes all the resources a ResourcePlacement selected.
```

Comment:
Hi Ryan! This is about line 61; `v1beta1.NamespacedName` requires a mandatory namespace, but for cluster-scoped resources we might need to drop that requirement I assume?